### PR TITLE
added Newpage

### PR DESCRIPTION
**Pull request by PriyaHirapara**

### CODE REVIEW

**Comment by PriyaHirapara** [comment]
hello

**Comment by PurviShah06** [comment]
bolo

File: src/Newpage.java [hunk 1]
```diff
@@ -0,0 +1,2 @@
+public class Newpage {
```
Comment (PurviShah06) [inline src/Newpage.java:1]: pls Add some method

Comment (PriyaHirapara) [reply src/Newpage.java:1]: thank for your quick review

**Comment by PurviShah06** [comment]
submit when u done

File: src/Newpage.java (hunk 1 above)
Comment (PurviShah06) [reply src/Newpage.java:1]: submit when u done

**Pull request merged**